content/jest.setupFilesAfterEnv.ts

```diff
@@ -0,0 +1,18 @@
+import * as logger from '@well-known-components/logger'
+
+beforeAll(() => {
+  // Mock logger implementation
+  const createLogComponent = logger.createConsoleLogComponent
+  jest.spyOn(logger, 'createLogComponent').mockImplementation(() => {
+    const logComponentMock = createLogComponent()
+    const loggerMock = logComponentMock.getLogger('__test__')
+    loggerMock.debug = () => {}
+    loggerMock.error = () => {}
+    loggerMock.info = () => {}
+    loggerMock.log = () => {}
+    loggerMock.warn = () => {}
+    logComponentMock.getLogger = jest.fn(() => loggerMock)
+    return logComponentMock
+  })
+  jest.spyOn(process.stdout, 'write').mockImplementation(() => true)
```

Comment:
Why is this necessary also?

Reply:
On the same line as with the [logger mock](https://github.com/decentraland/catalyst/pull/847#discussion_r781440312) changes, this change is not ultimately necessary, but it aims to remove console logs done in the tests and improve readability of the CI logs.
What would you suggest in this case?